compose/Windows.md

```diff
@@ -0,0 +1,91 @@
+# Team Edition Server Installation on Windows
+
+## Preparing your Windows environment
+
+When running on Windows, you need to ensure that WSL is enabled. We also recommend using `podman` with `podman-compose`.
+
+> Note:
+If you run Windows on a virtual machine, please ensure that your host machine supports nested virtualization.
+
+> Hint:
+If using Azure VMs, ensure that
+> - Your VM's `Security type` is set to `Standard`
+> - Its size supports nested virtualization. `D2as_v6` is known to work.
+
+Steps to prepare your Windows environment:
+
+1. Ensure your machine has WSL installed by executing `wsl.exe --status` in PowerShell or Command Prompt. If WSL is not installed,
+   follow the instructions that this command produces.
+2. Ensure the dependencies are installed. We'll need `Git`, `podman`, and `podman-compose`.
```

Comment:
We need instructions and/or links to git/prodman/python/pip/etc distributions.
The best way is to provide cmd script which will install all of this automatically.